package.json

```diff
@@ -106,5 +106,8 @@
     "typescript": "4.9.4",
     "wait-on": "7.0.1"
   },
+  "resolutions": {
+    "@graasp/sdk": "0.10.0"
+  },
```

Comment:
Still needed?

Reply:
until we bump the version in query-client and ui I think yes